windows_containers/windows-containers-release-notes-4-x.adoc

```diff
@@ -28,6 +28,18 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support. 
 endif::openshift-origin[]
 
+[id="wmco-4-0-2"]
+== Release notes for Red Hat Windows Machine Config Operator 4.0.2
+
+Issued: 2023-4-17
+
+The WMCO 4.0.2 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/.
+
+[id="wmco-4-0-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, the containers on the Windows nodes were assigned a hard-coded DNS server IP address (such as 172.30.0.10) that was based on the default subnet. This caused DNS resolution to fail in certain situations. This fix removes the hard-coded DNS address. Windows containers are now assigned a valid DNS server IP address that is based on the cluster DNS configuration. As a result, DNS resolution works as expected for Windows containers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020350[**BZ#2020350**])
```

Comment:
Just a minimalism suggestion to remove some of the extra articles. Up to you if you would like to take them. 
```suggestion
* Previously, containers on Windows nodes were assigned a hard-coded DNS server IP address (such as 172.30.0.10) based on the default subnet. This caused DNS resolution to fail in certain situations. This fix removes the hard-coded DNS address, and Windows containers are now assigned a valid DNS server IP address based on the cluster DNS configuration. As a result, DNS resolution works as expected for Windows containers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020350[**BZ#2020350**])
```

Reply:
Also, I wasn't sure about "[**BZ#2020350**])" should this be two * or should it be "[*BZ#2020350*])"

I see in 4.0.1 it is listed with two so disregard if it is supposed to be that way.